src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,12 @@
+export const ArticleCard = ({ article }) => {
+  return (
+    <li className="article-card">
+      <h3>{article.title}</h3>
+      <img src={article.article_img_url} alt={article.title} />
+      <p>Topic: {article.topic}</p>
+      <p>By: {article.author}</p>
+      <p>Votes: {article.votes}</p>
+      <p>Comments: {article.comment_count}</p>
+    </li>
```

Comment:
A bit picky, but would love the created_at also added here and would be interested to see how you choose to display this 😄 It will be handy for it you add a sort by feature based on date created!